layers/+tools/writeroom/packages.el

```diff
@@ -0,0 +1,26 @@
+;;; packages.el --- writeroom Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2015 Gergely Nagy
+;;
+;; Author: Gergely Nagy <algernon@madhouse-project.org>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(setq writeroom-packages
+      '(
+        writeroom-mode
+        ))
+
+(defun writeroom/init-writeroom-mode ()
+  "Initialize writeroom-mode"
+
+  (use-package writeroom-mode
+    :init (progn
+            (defun spacemacs/writeroom-mode ()
+              (interactive)
+
+              (spacemacs/set-leader-keys "aW" #'writeroom-mode))
+            (spacemacs/set-leader-keys "aW" #'spacemacs/writeroom-mode))))
```

Comment:
I don't know why you are defining a function on `SPC a W` that has for goal to defines another function on `SPC a W` :-)